### DISCUSSION TITLE
[REQUEST]Code snippets in README

### DISCUSSION THREAD

**Discussion by Zengyf-CVer** [Textualize/rich#3187] [General]
Many of my open source projects use your project. I am very interested in this picture in your README. I know how to make a flat code piece, but how to make a code piece with a certain rotation angle?
![image](https://github.com/Textualize/rich/assets/41098760/6b6f8d06-b423-46ee-9bcd-42825bef6532)

**Comment by willmcgugan** [comment]
Photoshop!
